src/settings.cpp

```diff
@@ -1285,6 +1286,7 @@ static void HandleSettingDescs(IniFile &generic_ini, IniFile &private_ini, IniFi
 		proc_list(private_ini, "server_bind_addresses", _network_bind_list);
 		proc_list(private_ini, "servers", _network_host_list);
 		proc_list(private_ini, "bans", _network_ban_list);
+		proc_list(secrets_ini, "server_authorized_keys", _settings_client.network.server_authorized_keys);
```

Comment:
Should we put this in `private`, and not in `secret`? It doesn't actually contain secrets. And it is pretty similar to "bans" in that regard.

Bit bikeshedding, I am aware.